prepare/templates/regression/bounded/pair.py

```diff
@@ -0,0 +1,24 @@
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.templates import OutputQuantizingTemplate, TemplatesList
+
+add_to_catalog(
+    OutputQuantizingTemplate(
+        input_format="""
+                   Given this text: '{text1}' and this text: '{text2}', on a scale of {min_value} to {max_value}, what is the {type_of_value} of this texts?
```

Comment:
Why not keep to existing format?

Given this sentence: '{text1}', on a scale of {min_value} to {max_value}, what is the {type_of_relation} to this text  {text2}